### ISSUE TITLE
gh access token not required (?)

### ISSUE THREAD

**Issue by klieret** [savvy379/codashep_ml_2022#1]
Maybe I'm missing something, but why would one need to create a gh access token to simply clone a public (!) repository? Your instructions also don't tell them to fork it, so they would not have write permissions there either...

I think we can remove this part of the setup :)

**Comment by neutralino-ai** [comment]
also a bit confusing how to open the notebook from colab after downloading.. seems colab can only read fiels under `Colab Notebooks` folder. In this tutorial files are downloaded in the `Githubs` folder and they are invisible.

**Comment by klieret** [comment]
Hmm, that worked for me though. I was just navigating to the `Github` folder on my google drive and then double-clicking and they opened in Colab. Does that work for you?

**Comment by neutralino-ai** [comment]
> 

No, nothing appears there.. I can only download them, but cannot run them

<img width="1160" alt="image" src="https://user-images.githubusercontent.com/3205938/182451438-29655285-d39c-4de1-92fb-ccffebf82823.png">
<img width="430" alt="image" src="https://user-images.githubusercontent.com/3205938/182451515-70b03cce-8b5a-41e6-8ed2-4c6db919577f.png">

**Comment by neutralino-ai** [comment]
Problem solved:

The python notebook files should not be opened in the `colab.research.google.com` through the "file" menu. Instead, they should be opened in the google drive website. The files will be automatically opended with colab.
